dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -664,13 +666,13 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
     private Set<String> getAllowedFFDCAnnotationFromTest(FrameworkMethod m) {
 
         Set<String> annotationListPerClass = new HashSet<String>();
-
         // Method
-        Set<AllowedFFDC> ffdcs = new HashSet<AllowedFFDC>();
-        ffdcs.add(m.getAnnotation(AllowedFFDC.class));
+        AllowedFFDC[] allowedffdcs = m.getMethod().getAnnotationsByType(AllowedFFDC.class);
```

Comment:
Why isn't the logic here the same as ExpectedFFDC?

Reply:
The difference between the original ExpectedFFDC (which is restricted to @Target({ ElementType.METHOD }))and AllowedFFDC annotations is the reason for the difference in logic here and I did not alter it.